### PR TITLE
[PLAT-19]

### PR DESCRIPTION
**Pull request by Morphasis**
- updated naming convention to be var_var funInCamel to be consistent
- time on current_item changed method so it is only called on the continue button.

### CODE REVIEW

File: src/public-booking/templates/event_list.html [hunk 1]
```diff
@@ -168,7 +168,7 @@ <h3 class="truncate-sm">{{item.description}}</h3>
                       <span class="fa fa-clock-o" aria-hidden="true"></span>
                       <span ng-bind="item.date.format('h[:]mma')"></span>
                       <span class="hidden-sm" ng-class="{'hidden-lg':$root.tiles, 'hidden-md':$root.tiles}">-</span>
-                      <span class="hidden-sm" ng-class="{'hidden-lg':$root.tiles, 'hidden-md':$root.tiles}" ng-bind="item.end_datetime.format('h[:]mma')"></span>
+                      <span class="hidden-sm" ng-class="{'hidden-lg':$root.tiles, 'hidden-md':$root.tiles}" ng-bind="item.endDatetime.format('h[:]mma')"></span>
```
Comment (lukeferguson) [inline src/public-booking/templates/event_list.html]: `endDatetime` is a method if it's notation is correct

**Comment by Morphasis** [comment]
I need to update the cal template before merging this will update shortly

**Comment by Morphasis** [comment]
I BROKE THIS PLEASE DO NOT MERGE JUST YET

File: src/public-booking/templates/checkout_event.html [hunk 2]
```diff
@@ -10,7 +10,7 @@ <h1 class="truncate-sm">{{bb.event_details.name}}</h1>
         </li>
         <li class="col-xs-6 col-sm-3 time truncate-xs">
           <span class="glyphicon glyphicon-time" aria-hidden="true"></span>
-          <span class="event-time" ng-bind="bb.event_details.datetime.format('h[:]mma')"></span>&ndash;<span ng-bind="bb.event_details.end_datetime.format('h[:]mma')"></span>
+          <span class="event-time" ng-bind="bb.event_details.datetime.format('h[:]mma')"></span>&ndash;<span ng-bind="bb.event_details.endDatetime.format('h[:]mma')"></span>
```
Comment (lukeferguson) [inline src/public-booking/templates/checkout_event.html]: @Morphasis endDateTime is a method! Did you test this?

File: src/public-booking/templates/event.html [hunk 3]
```diff
@@ -12,7 +12,7 @@ <h1 class="truncate-sm">{{event.chain.name}}</h1>
           </li>
           <li class="col-xs-6 col-sm-4 time truncate-xs">
             <span class="fa fa-clock-o" aria-hidden="true"></span>
-            <span class="event-time" ng-bind="event.date.format('h[:]mma')"></span>&ndash;<span ng-bind="event.end_datetime.format('h[:]mma')"></span>
+            <span class="event-time" ng-bind="event.date.format('h[:]mma')">&ndash;<span ng-bind="event.endDatetime.format('h[:]mma')"></span>
```
Comment (lukeferguson) [inline src/public-booking/templates/event.html]: @Morphasis endDateTime is a method! Did you test this?

File: src/core/javascripts/purchase/controllers/purchase.js.coffee [hunk 4]
```diff
@@ -177,7 +177,7 @@ angular.module('BB.Controllers').controller 'Purchase', ($scope,  $rootScope, Co
         $scope.bb.moving_booking = booking
         $scope.quickEmptybasket()
         new_item = new BBModel.BasketItem(booking, $scope.bb)
-        new_item.setSrcBooking(booking, $scope.bb)
```
Comment (lukeferguson) [inline src/core/javascripts/purchase/controllers/purchase.js.coffee]: Why did you remove $scope.bb @Morphasis?
